package.json

```diff
@@ -19,7 +19,7 @@
   "dependencies": {
     "chalk": "^3.0.0",
     "xml2js": "^0.4.23",
-    "yeoman-generator": "^4.5.0",
+    "yeoman-generator": "^5.4.0",
```

Comment:
@ameyanator have you validated all the scenarios with this plugin, like create app/delete app/deploy/build etc?

Reply:
I validated create and delete. I'll check for deploy and build too